mix.exs

```diff
@@ -61,8 +61,8 @@ defmodule Wallaby.Mixfile do
       {:credo, "~> 0.9", only: [:dev, :test], runtime: false},
       {:bypass, "~> 1.0.0", only: :test},
       {:excoveralls, "~> 0.7", only: :test},
```

Comment:
Does ecto_sql need to be listed as an optional dependency? That might take care of some of those dialyzer errors as well...

Reply:
Do we still have many dialyzer issues? Do they show up in the library or in tests that are using wallaby?

I just tried running dialyzer on this branch. It only found one issue in one of the tests where `:ok` return type should have been `true`

Reply:
It would be if we add some phoenix/ecto integration tests.